model/message.go

```diff
@@ -62,4 +63,11 @@ type MessageExternal struct {
 	// required: true
 	// example: 2018-02-27T19:36:10.5045044+01:00
 	Date time.Time `json:"date"`
+	// The tokens of the clients.
+	//
+	// read only: true
+	// required: false
+	// example: ["token1","token2","token3"]
+	// if empty, the message is sent to all clients
+	Clients[] string `form:"clients[]" query:"-" json:"clients"`
```

Comment:
I don't think its a good idea to refer to the clients by their token as applications are not supposed to know this formation, and it would be inappropriate to send other (possibly admin) client tokens to all clients. I would say use client id instead.